KIPs/kip-7-fungible_token.md

```diff
@@ -26,399 +26,229 @@ A standard interface allows any token on Klaytn to be re-used by other applicati
-clients SHOULD make sure to create user interfaces in such a way that they set the allowance first to `0` before setting it to another value for the same spender.
-THOUGH The contract itself shouldn't enforce it, to allow backwards compatibility with contracts deployed before.
-
-```solidity
-function approve(address _spender, uint256 _value) public returns (bool success)
-```
-
-
-#### allowance
-
-Returns the amount which `_spender` is still allowed to withdraw from `_owner`.
-
-```solidity
-function allowance(address _owner, address _spender) public view returns (uint256 remaining)
-```
-
-#### name
-
-Returns the name of the token - e.g. `"MyToken"`.
-
-OPTIONAL - This method can be used to improve usability,
-but interfaces and other contracts MUST NOT expect these values to be present.
-
-
-```solidity
-function name() public view returns (string)
-```
-
-
-#### symbol
-
-Returns the symbol of the token. - e.g. `"MT"`.
-
-OPTIONAL - This method can be used to improve usability,
-but interfaces and other contracts MUST NOT expect these values to be present.
-
-```solidity
-function symbol() public view returns (string)
-```
-
-
-#### decimals
-
-Returns the number of decimals the token uses - e.g. `8`, means to divide the token amount by `100000000` (10^8) to get its user representation.
-
-OPTIONAL - This method can be used to improve usability,
-but interfaces and other contracts MUST NOT expect these values to be present.
-
-```solidity
-function decimals() public view returns (uint8)
-```
-
-#### mint
-
-Mints `_value` amount of tokens to address `_to`, and MUST fire the [Transfer event](#transfer-1). The value of `_from` MUST be set to `0x0` in the transfer event.
-The function SHOULD `throw` if the `_to` is `0x0`.
-The function SHOULD `throw` if the message sender is not a minter.
-
-OPTIONAL - This method can be used to improve usability,
-but interfaces and other contracts MUST NOT expect these values to be present.
-
-```solidity
-function mint(address _to, uint256 _value) public returns (bool)
-```
-
-#### isMinter
-
-Returns `true` if `_account` is a minter or `false` if `_account` is not a minter. 
-
-```solidity
-function isMinter(address _account) public view returns (bool)
-```
-
-#### addMinter
-
-Adds `_account` as a minter, and MUST fire the [MinterAdded event](#minteradded). The value of `_account` MUST be set with same account of `addMinter` method in the `MinterAdded` event.
-The function SHOULD `throw` if the `_account` is already a minter.
-The function SHOULD `throw` if the message sender is not a minter.
-
-OPTIONAL - This method can be used to improve usability,
-but interfaces and other contracts MUST NOT expect these values to be present.
+|[IKIP7](#kip7-interface)|0x36372b07|
+|[IKIP7Metadata](#metadata-extension)|0xa219a025|
+|[IKIP7Mint](#mint-extension)|0xeab83e20|
```

Comment:
How about matching the KIP-17 with the interface name?
KIP-17 is naming extension interface names in this way "~able".

Reply:
Reflected. Thanks.